### ISSUE TITLE
Sometimes "ERROR" 3000 in HMS 5.7 File move failing

### ISSUE THREAD

**Issue by berg128** [hmailserver/hmailserver#359]
I have seen below error a few times in my logs:

"ERROR"	3000	"2021-01-23 07:22:43.251"	"Severity: 2 (High), Code: HM5049, Source: File::Normal, Description: Could not move the file D:\Program Files\hMailServer\Data\{64490858-0CAB-4014-9E0F-E6CE413AF406}.eml.tmp to D:\Program Files\hMailServer\Data\{64490858-0CAB-4014-9E0F-E6CE413AF406}.eml. Tried 5 times without success., Error code: 5, Message: Access is denied"

The mail is normally delivered so problems with that. The .tmp file is left in the data folder and all I have to do is remove it from there.

**Comment by RvdHout** [comment]
Can't say i have seen that error on my 5.7.x hMailServer test instance

Are you sure this isn't a permission issue? I noticed the non-default path D:\Program Files\hMailServer\
Maybe check if D:\Program Files\hMailServer\ has sufficient permission set for the user hMailService is running under

**Comment by berg128** [comment]
I figured it must have something to do with a conflict between SA and ClamAv. I use Clamav option in HMS and also the ClamAV plugin and filescan. The pugin was with version 2.0 but I must have used an old link for the filescan module from https://www.hmailserver.com/forum/viewtopic.php?f=22&t=33101&p=224083&hilit=spamassassin+plugin#p224083 because it turned out to be 1.6. I replaced with 1.8 now and will keep you posted.
